src/context/yaml/index.ts

```diff
@@ -18,12 +18,14 @@ export default class YAMLContext {
   mappings: KeywordMappings;
   mgmtClient: Auth0APIClient;
   assets: Assets;
+  disableKeywordReplacement: boolean;
 
   constructor(config: Config, mgmtClient) {
     this.configFile = config.AUTH0_INPUT_FILE;
     this.config = config;
     this.mappings = config.AUTH0_KEYWORD_REPLACE_MAPPINGS || {};
     this.mgmtClient = mgmtClient;
+    this.disableKeywordReplacement = false;
```

Comment:
By default, keyword replacement is enabled (not disabled). This configuration value is only enabled during the keyword preservation stage.